### DISCUSSION TITLE
GitHub Education Benifits Not Available!

### DISCUSSION THREAD

**Discussion by bobbyhan11** [community/community#141260] [GitHub Education]
### Hi everyone,

My educational discount request has been approved since 24.10.10, but I don't receive my GitHub Education Benifits!

**Comment by davevad93** [comment]
Hi @Bobby-Han ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Bobby-Han was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by NDSinChina** [comment]
I am facing the same issue @davevad93

**Reply by League2EB** [reply]
I am facing the same issue @davevad93

**Reply by davevad93** [reply]
Hi @NDSinChina and @League2EB don't worry, as stated above, the waiting time may vary and can take up to a couple of weeks.
